nengo/spa/assoc_mem.py

```diff
@@ -60,8 +54,9 @@ def __init__(self, input_vocab, output_vocab=None,  # noqa: C901
                  default_output_key=None, threshold=0.3,
                  inhibitable=False, wta_output=False,
                  wta_inhibit_scale=3.0, wta_synapse=0.005,
```

Comment:
Can we or would we want to modify this to use the parameter/config system, so that we can set default values for these things?